.github/workflows/pr-artifacts.yml

```diff
@@ -0,0 +1,37 @@
+name: pr-artifacts
+
+on:
+  workflow_run:
+    workflows: [generate-builds]
+    branches-ignore: [develop, develop-zhora]
+    types:
+      - completed
+
+jobs:
+  pr-artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - id: 'get-info'
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: context.payload.workflow_run.id,
+            });
+
+            return allArtifacts.data.artifacts.reduce((acc, item) => {
+              if (item.name === "assets") return acc;
+              acc += `
+              - [${item.name}](${item.archive_download_url})`;
+              return acc;
+            }, '### Build Artifacts');
+      - id: 'add-to-pr'
+        uses: garrettjoecox/pr-section@3.1.0
+        with:
```

Comment:
Is this something we can host on the HarbourMasters side instead? I'm not quite sure how these scripts work.

Reply:
We could fork it if necessary, but it’s something that I created a few years back and use for various things, not really specific to HM

Reply:
The reason why I asked is because I'm not sure if depending on someone else's repo for anything that's used in this repo is a good idea. I'll leave that up to @dcvz or @briaguya-ai.

Reply:
Right, sounds good to me then.